client/my-sites/plan-features-2023-grid/components/plan-features.tsx

```diff
@@ -0,0 +1,103 @@
+import { JetpackLogo } from '@automattic/components';
+import { useDesktopBreakpoint } from '@automattic/viewport-react';
+import { LocalizeProps } from 'i18n-calypso';
+import { useState, useEffect } from 'react';
+import PlanFeatures2023GridFeatures from '../components/features';
+import { PlanProperties, DataResponse } from '../types';
+import { Plans2023Tooltip } from './plans-2023-tooltip';
+import type { DomainSuggestion } from '@automattic/data-stores';
+
+const PlanFeatures: React.FC< {
+	planProperties: PlanProperties[];
+	paidDomainName?: string;
+	wpcomFreeDomainSuggestion: DataResponse< DomainSuggestion >; // used to show a wpcom free domain in the Free plan column when a paid domain is picked.
+	translate: LocalizeProps[ 'translate' ];
+	hideUnavailableFeatures?: boolean; // used to hide features that are not available, instead of strike-through as explained in #76206
+	selectedFeature?: string;
+	isCustomDomainAllowedOnFreePlan: DataResponse< boolean >; // indicate when a custom domain is allowed to be used with the Free plan.
+	isTableCell: boolean | undefined;
+	Container: (
+		props: React.HTMLAttributes< HTMLDivElement > | React.HTMLAttributes< HTMLTableCellElement >
+	) => JSX.Element;
+} > = ( {
+	planProperties,
+	paidDomainName,
+	wpcomFreeDomainSuggestion,
+	translate,
+	hideUnavailableFeatures,
+	selectedFeature,
+	isCustomDomainAllowedOnFreePlan,
+	isTableCell,
+	Container,
+} ) => {
+	const [ activeTooltipId, setActiveTooltipId ] = useState( '' );
+	const isDesktop = useDesktopBreakpoint();
+
+	useEffect( () => {
```

Comment:
This file holds the features as a separate child component which previously existed in client/my-sites/plan-features-2023-grid/index.tsx. The addition here is the tooltip state handler and useEffect to determine the currently active tooltip.